etc/sdk.api.md

```diff
@@ -217,6 +217,15 @@ export interface ListDirectoryResponse {
     items: DirectoryEntry[];
 }
 
+// @public (undocumented)
+export interface MakeFilePublicRequest {
+    // (undocumented)
+    bucket: string;
+    // (undocumented)
+    path: string;
+    writable: boolean;
+}

```

Comment:
Also making a note, when we add the sharing pieces, we might need to add dbId as a param here.